sigs.yaml

```diff
@@ -1,4 +1,42 @@
 sigs:
+- dir: sig-community-experience
+  name: Community Experience
+  mission_statement: >
+    Marketing/Communications/Org-Ops/Governance/Community owns O2KR2 “Communications”
+    and “Metrics” owns O2KR1 “all workstreams / people”
+
+  charter_link: charter.md
+  label: community-experience
+  leadership:
+    chairs:
+    - github: codificat
+      name: Pep Turró Mauri
+      company: Red Hat
+    - github: schwesig
+      name: Thorsten Schwesig
+      company: Red Hat
+  meetings:
+  - description: Community Experience SIG meeting
+    day: Wednesday
+    time: "17:00"
+    tz: CET (Central Europe Time)
+    frequency: bi-weekly
+    url: https://meet.google.com/tig-yuxq-fyh
+    archive_url: https://docs.google.com/document/d/1t47S9JUZbI3q0_LgVtaBKYsaGqnOP6YSG6x8XaBhiew/edit
+  contact:
+    liaison:
+      github: schwesig
+      name: Thorsten Schwesig
+  subprojects:
+  - name: ComCom
+    owners:
+    - https://raw.githubusercontent.com/open-services-group/community/main/sig-community-experience/OWNERS
+  - name: GovOrg
+    owners:
+    - https://raw.githubusercontent.com/open-services-group/community/main/sig-community-experience/OWNERS
+  - name: Metrics Strategic
+    owners:
+    - https://raw.githubusercontent.com/open-services-group/community/main/sig-community-experience/OWNERS
```

Comment:
We should think about a way to point to the real owners/members of each sub-project.

For the Metrics sub-project, we should also clarify the relationship with the [Metrics sub-project in the data science SIG](https://github.com/open-services-group/community/tree/main/sig-data-science#metrics-1).

Reply:
opened an issue https://github.com/open-services-group/community/issues/66 for this to take care of it.